src/components/shared/header-component/Header/DesktopHeader.jsx

```diff
@@ -25,7 +26,8 @@ const DesktopHeader = ({ intl }) => {
   const allowPublicAccountCreation = getConfig().ALLOW_PUBLIC_ACCOUNT_CREATION ?? true;
```

Comment:
Please delete this unused const after double check

Reply:
Right. Will remove.